congruity/congruity.py

```diff
@@ -1863,8 +1863,14 @@ def main():
             else:
                 raise CmdLineException("ERROR: Option '%s' not recognized" % arg)
         if len(argv) != 1:
-            raise CmdLineException("ERROR: Precisely one filename argument is required")
-        ezhex_filename = argv.pop(0)
+            # We did not get a file name on the command line, prompt the user for one.
+            with wx.FileDialog(None, "Congruity - Open a file", wildcard="EZHex files (*.ez*)|*.ez*",
```

Comment:
Unfortunately, this doesn't work on Linux as EZHex files are named <abc>.EZHex (case sensitive).

Reply:
Fixed, good catch!